### PR TITLE
chore: session replay project config for mobile - what is possible

### PR DESCRIPTION
**Pull request by marandaneto**
## Problem

Sometimes customers get confused about whats possible or not yet via the project config, so let's add a hint about what is available or not for Mobile, so they don't waste time testing something that wouldn't work yet.
We already have open issues for those controls and we will eventually implement all of them.

## Changes

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

### CODE REVIEW

**Review by ioannisj — APPROVED** [review]
Just left a few minor comments. I think this will help a lot!
I'm wondering if it's also worth on working on something similar to https://posthog.com/docs/libraries but with sub-features as well

File: frontend/src/scenes/settings/environment/SessionRecordingIngestionSettings.tsx [hunk 1]
```diff
@@ -335,6 +337,7 @@ function EventTriggerOptions(): JSX.Element | null {
                 Session recording will be started immediately before PostHog queues any of these events to be sent to
                 the backend.
             </p>
```
[the remark this answers is not in the document]
Comment (marandaneto) [reply frontend/src/scenes/settings/environment/SessionRecordingIngestionSettings.tsx:339]: It's always below right now.

File: frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx [hunk 2]
```diff
@@ -24,6 +24,13 @@ function LogCaptureSettings(): JSX.Element {
                 This setting controls if browser console logs will be captured as a part of recordings. The console logs
                 will be shown in the recording player to help you debug any issues.
             </p>
+            <p>
+                Log capture is also available for{' '}
+                <Link to="https://posthog.com/docs/session-replay/mobile" target="_blank">
+                    Mobile session replay
```
[the remark this answers is not in the document]
Comment (marandaneto) [reply frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx:30]: true, I don't think we need to go too granular here, but we can if users still get confused with that.

[the remark this answers is not in the document]
Comment (marandaneto) [reply frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx:30]: https://github.com/PostHog/posthog-ios/issues/258 just so we dont forget

**Comment by pauldambra** [comment]
so we have

| Feature                    | Web | iOS | Android | Flutter | React Native |
|----------------------------|-----|-----|---------|---------|--------------|
| Console log capture        |     |     |         |         |              |
| Canvas capture (obvs web only) |  ✔  |     |         |         |              |
| Network timing capture     |     |     |         |         |              |
| Network header capture     |     |     |         |         |              |
| Network body capture       |     |     |         |         |              |
| Authorized domains         |     |     |         |         |              |
| Sampling                   |     |     |         |         |              |
| Minimum duration           |     |     |         |         |              |
| Linked flag                |     |     |         |         |              |
| URL trigger                |     |     |         |         |              |
| URL denylist               |     |     |         |         |              |
| Event trigger              |     |     |         |         |              |

feels like we could have a nice row of tags alongside the headers showing what's supported giving somewhere for a tooltip or something to highlight minimum versions?

(great idea btw)

File: frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx [hunk 3]
```diff
@@ -197,6 +205,13 @@ export function NetworkCaptureSettings(): JSX.Element {
                     />
                 </div>
             </div>
+            <p>
+                Network capture is also available for{' '}
```
[the remark this answers is not in the document]
Comment (marandaneto) [reply frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx]: its not web only, its on mobile too but via code, on iOS (captureNetworkTelemetry)

**Comment by marandaneto** [comment]
> So we have
> 
> Feature	Web	iOS	Android	Flutter	React Native
> Console log capture					
> Canvas capture (obvs web only)	✔				
> Network timing capture					
> Network header capture					
> Network body capture					
> Authorized domains					
> Sampling					
> Minimum duration					
> Linked flag					
> URL trigger					
> URL denylist					
> Event trigger					
> feels like we could have a nice row of tags alongside the headers showing what's supported giving somewhere for a tooltip or something to highlight minimum versions?
> 
> (great idea btw)

yep a good idea, but I think it'd bloat the UI even more, maybe this would make more sense in the documentation and the project settings just link to that section.
I'd like to get this PR merged though since its a quick improvement already on what's available and what's not on Mobile.
is that ok?

**Comment by pauldambra** [comment]
> I'd like to get this PR merged though since its a quick improvement already on what's available and what's not on Mobile.
is that ok?

yep... my guess is people don't actually read this text 🙈 but fine to merge this and come back to it

making the skeleton for the table made me realise I don't know the answer to which boxes should be ticked... might be worth us having this tick table somewhere so we can track the gaps??

**Comment by marandaneto** [comment]
I filled the table with what I know by heart, should be correct but feel free to double check

File: frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx [hunk 4]
```diff
@@ -24,6 +24,13 @@ function LogCaptureSettings(): JSX.Element {
                 This setting controls if browser console logs will be captured as a part of recordings. The console logs
                 will be shown in the recording player to help you debug any issues.
             </p>
+            <p>
+                Log capture is also available for{' '}
+                <Link to="https://posthog.com/docs/session-replay/mobile" target="_blank">
```
Comment (marandaneto) [inline frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx:29]:
```suggestion
                <Link to="https://posthog.com/docs/session-replay/console-log-recording" target="_blank">
```

**Pull request merged**